src/dftbp/dftbplus/inputdata.F90

```diff
@@ -552,6 +552,33 @@ module dftbp_dftbplus_inputdata
     !> Hybrid xc-functional input
     type(THybridXcInp), allocatable :: hybridXcInp
 
+
+    !> Multipole expansion
```

Comment:
Probably should be a derived type. Also, can some of these be combined into matrices of quantities instead of separate variables for x,y and z ?